pinc/archiving.inc

```diff
@@ -64,9 +75,13 @@ function archive_project($project, $dry_run = false)
             echo "    Move $project_dir to $new_dir.\n";
         } else {
             // Remove uncompressed versions of whole-project texts, leaving zips.
-            exec("rm $project_dir/projectID*.txt");
+            $process = new Process(["rm", "$project_dir/projectID*.txt"]);
+            $process->run();
+            if (!$process->isSuccessful()) {
+                throw new RuntimeException(sprintf(_("Unable to delete %s"), "$project_dir/projectID*.txt"));
+            }
```

Comment:
This change is slightly problematic because there might not be files matching that format to delete which throws an error. We need to add a `glob()` to see if there are any files before trying to delete them.

```suggestion
            if (glob("$project_dir/projectID*.txt")) {
                $process = new Process(["rm", "$project_dir/projectID*.txt"]);
                $process->run();
                if (!$process->isSuccessful()) {
                    throw new RuntimeException(sprintf(_("Unable to delete %s"), "$project_dir/projectID*.txt"));
                }
            }

```